### PR TITLE
Bundles Landing Page - Copy Changes Episode II

### PR DESCRIPTION
**Pull request by Ap0c**
## Why are you doing this?

Had to open a new PR for #268 because of a rebase after closure.

We want the copy to say slightly different things before we launch in the UK. This PR updates the bundles landing page to incorporate those changes.

[**Trello Card**](https://trello.com/c/U7WR2JsC/952-update-bundles-landing-page-language-structure)

cc: @patstirling @amohkhan @elvisr

## Changes

- Added a new `crossProduct` component for the new benefit.
- Tweaked the copy across:
  + Introduction text
  + Lists of bundle features (inc. for patrons and events)
  + Some of the CTAs
  + The why support paragraphs
- Added some copy to the contributions bundle.
- Dropped the hero image for the why support section.
- Removed `IntroductionText` shared component, pulled the code into the `Introduction` component.
- Removed the bundles bleed, the background for the introduction and bundles is now the same.
- Changed some colours.

## Screenshots

|  Desktop | Tablet  |  Mobile |
|---|---|---|
| tweaked-desktop  |  tweaked-tablet | tweaked-mobile |

### CODE REVIEW

File: assets/pages/bundles-landing/components/Bundles.jsx [hunk 1]
```diff
@@ -269,6 +272,7 @@ function Bundles(props: PropTypes) {
 // ----- Map State/Props ----- //
 
 function mapStateToProps(state) {
+  console.log(state);
```
Comment (svillafe) [inline assets/pages/bundles-landing/components/Bundles.jsx]: I would delete this console.log before going live.

File: assets/pages/bundles-landing/components/crossProduct.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+
+// ----- Component ----- //
+
+export default function CrossProduct() {
```
Comment (rupertbates) [inline assets/pages/bundles-landing/components/crossProduct.jsx]: I thought this was going to be the mathematical usage of cross product!

**Comment by JustinPinner** [comment]
Looks ok to me 👍 ✍️ 🚢

**Comment by Ap0c** [comment]
Some repeated flakiness, but it looks ok ^.

**Pull request merged**